GVFS/GVFS.Virtualization/Background/FileSystemTask.cs

```diff
@@ -21,6 +21,7 @@ public enum OperationType
             OnFileOverwritten,
             OnFileSuperseded,
             OnFileConvertedToFull,
+            OnFailedFileHydration,

```

Comment:
`OnFailedFileHydration` needs to be added to the end of the list.  

`OperationType` values are persisted, and if we add new values in the middle it could break things when VFS4G loads values from disk.

Could you add a comment that new values should be added to the end?  It isn't obvious looking at this code that we need to.